CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@
 
 The Code of Conduct explains the *bare minimum* behavior
 expectations the Node Foundation requires of its contributors.
-[Please read it before participating.](https://github.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md)
+[Please read it before participating.](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
```

Comment:
I think the CoC has just been moved in the admin repo: https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md
https://github.com/nodejs/admin/pull/25

Reply:
@ajmeese7 can you please update this pr ?

Reply:
@daxlab I accidentally deleted the merging branch so I'm not sure how to do that